source/locale/fr/symbols.dic

```diff
@@ -40,7 +40,7 @@ symbols:
 
 decimal point		# virgule décimale
 thousands separator		# point des mille
-dates .	\1 \2 \3 	all	always	# point de date
+dates .	\1 \2 \3 	all	norep	# point de date
```

Comment:
@sthibaul Interesting to note that this dates replacement is not done in the English dictionary! 
@josephsl Will this modification cause a problem for translators?

Reply:
As I mentioned previously, this is a bug in the french translation, I'm discussing with the translator, so we get that included before this gets commited (without having to do the change, then, since it'll already be there).

Reply:
I see. I'll add the blocked label to this PR so that it isn't merged until the translation change comes through. Please add a comment when that happens.